main.md

```diff
@@ -918,6 +918,60 @@ resource). The effect of this likely simplifies the logistics of phased
 upgrades to protected resources in their support DPoP or even 
 prolonged deployments of protected resources with mixed token type support. 
 
+If a protected resource supporting both `Bearer` and `DPoP` schemes elects to
+respond with multiple `WWW-Authenticate` challenges, attention should be paid to
+which challenge(s) should deliver the actual error information. It is
+RECOMMENDED that the following rules be adhered to:
+
+* If no authentication information has been included with the request, then the
+challenges SHOULD NOT include an error code or other error information, as per
+[@!RFC6750], Section 3.1 ((#multi-challenge-no-token)).
+
+* If the mechanism used to attempt authentication could be established
+unambiguously, then the corresponding challenge SHOULD be used to deliver error
+information ((#multi-challenge-invalid-token)).
+
+* Otherwise, both `Bearer` and `DPoP` challenged MAY be used to deliver error
+information ((#multi-challenge-ambiguous)).
+
+!---
+```
+GET /protectedresource HTTP/1.1
+Host: resource.example.org
+
+HTTP/1.1 401 Unauthorized
+WWW-Authenticate: Bearer, DPoP algs="ES256 PS256"
+```
+!---
+Figure: HTTP 401 Response to a Protected Resource Request without Authentication {#multi-challenge-no-token}
+
+!---
+```
+GET /protectedresource HTTP/1.1
+Host: resource.example.org
+Authorization: Bearer INVALID_TOKEN
+
+HTTP/1.1 401 Unauthorized
+WWW-Authenticate: Bearer error="invalid_token",
+    error_description="Invalid token", DPoP algs="ES256 PS256"
+```
+!---
+Figure: HTTP 401 Response to a Protected Resource Request with Invalid Authentication {#multi-challenge-invalid-token}
+
+!---
+```
+GET /protectedresource HTTP/1.1
+Host: resource.example.org
+Authorization: Bearer Kz~8mXK1EalYznwH-LC-1fBAo.4Ljp~zsPE_NeO.gxU
+Authorization: DPoP Kz~8mXK1EalYznwH-LC-1fBAo.4Ljp~zsPE_NeO.gxU
+
+HTTP/1.1 400 Bad Request
+WWW-Authenticate: Bearer error="invalid_request", error_description="Multiple methods used to include an access token",
```

Comment:
This example runs over the width limit of 72 and needs to be shortened.  xml2rfc complains and the output looks bad.

Reply:
![Screen Shot 2022-12-15 at 4 50 20 PM](https://user-images.githubusercontent.com/3516573/207990664-ca37c3c3-6577-4b05-9f7d-6812e01682d3.png)

Reply:
![Screen Shot 2022-12-15 at 5 00 24 PM](https://user-images.githubusercontent.com/3516573/207991805-f650bd2d-5457-4bf6-a9f9-9b0923865fe8.png)

Reply:
I wasn't anticipating figures/examples with this to be honest. But they need to fit/format correctly, if included.

Reply:
Also add yourself to the acknowledgements, if you aren't already there. Which I think maybe you are. So never mind me blabbering here.